src/services/api-service.js

```diff
@@ -2,14 +2,14 @@
 import { WickrIOAPI } from '../helpers/constants'
 
 class APIService {
-  static getSecurityGroups() {
+  getSecurityGroups() {
```

Comment:
Why no longer static?

Reply:
because the commands are using this.apiService.getSecurityGroups. with static, it's not accessible directly